requirements.txt

```diff
@@ -15,3 +15,4 @@ straight.plugin
 psycopg2
 pytoml
 wtforms
+python-dateutil
```

Comment:
As mentioned above, you may find arrow easier to work with: https://arrow.readthedocs.io/en/latest

Reply:
I only need it to actually parse the ISO format of the date, even if it is in UTC.